web-client/public/js/graph.js

```diff
@@ -112,46 +124,71 @@ var drawGraph = function (nodes, links, positiveWeights, negativeWeights, sheetT
                      .append("g")
 
 
+  var previousValue = zoom.translate().slice();
+  var previousMax = [width, height];
+  var previousScale = zoom.scale();
   function zoomed(manual = false) {
```

Comment:
Woah, really? I didn't know that that was ES6 exclusive syntax! I'll change that to be ES5 then and do a manual workaround for it for now.